modules/oe_whitelabel_news/templates/node--oe-news--full.html.twig

```diff
@@ -0,0 +1,24 @@
+{#
+/**
+ * @file
+ * News full display.
+ */
+#}
+{% set badges = [] %}
```

Comment:
we need that? We can put badges: badges|render|default([])??

Reply:
Agreed it will clean the template and remove an unused array.